ksqldb-engine/src/main/java/io/confluent/ksql/util/SharedKafkaStreamsRuntimeImpl.java

```diff
@@ -63,9 +63,15 @@ public void register(
       final QueryId queryId
   ) {
     collocatedQueries.put(queryId, binpackedPersistentQueryMetadata);
-    log.info("Registered query in shared runtime: {}\n"
+    log.info("Registered query: {}  in {} \n"
                  + "Runtime {} is executing these queries: {}",
```

Comment:
This is a still a shared runtime, right?

Also, I'm tempted to suggest we remove the duplicated runtime id.  (I'm very much on the fence.)